3rdparty/pthreads4w/implement.h

```diff
@@ -38,9 +38,9 @@
 #if !defined(_IMPLEMENT_H)
 #define _IMPLEMENT_H
 
-#if !defined(PTW32_CONFIG_H)
-# error "config.h was not #included"
-#endif
+//#if !defined(PTW32_CONFIG_H)
+//# error "config.h was not #included"
+//#endif
```

Comment:
You should understand why this doesn't work instead of just commenting out code.

Reply:
I removed it to see if it would compile on here, same error unfortunately.

Reply:
The issue is fixed now, forgot to update an preprocessor definition in the VS project file.